src/GraphQL/loaders/defaultGraphQLTypes.js

```diff
@@ -1056,6 +1107,8 @@ const load = parseGraphQLSchema => {
   parseGraphQLSchema.graphQLTypes.push(POLYGON_CONSTRAINT);
   parseGraphQLSchema.graphQLTypes.push(FIND_RESULT);
   parseGraphQLSchema.graphQLTypes.push(SIGN_UP_RESULT);
+  parseGraphQLSchema.graphQLTypes.push(ElEMENT);
+  parseGraphQLSchema.graphQLTypes.push(ARRAY_RESULT);
```

Comment:
We will need to remember to include ArrayResult in the reserved names of the other PR